lib/fluent/plugin/out_splunk_hec.rb

```diff
@@ -51,12 +53,13 @@ class SplunkHECOutput < ObjectBufferedOutput
 
     def configure(conf)
       super
-      raise ConfigError, "'channel' parameter is required when 'use_ack' is true" if @use_ack && !@channel
+      raise ConfigError, "'channel' parameter is required when 'use_ack' is true" if @use_ack && !@auto_generate_channel && !@channel
```

Comment:
channel or auto_generate_channel parameters?

Reply:
auto generate is default false.so need to checkl

Reply:
I meant the error message.